README.md

```diff
@@ -102,13 +102,20 @@ To uninstall the chart:
 
 | Name                     | Description                                                                                  | Value           |
 | ------------------------ | -------------------------------------------------------------------------------------------- | --------------- |
-| deployment.probes.readinessProbe | The readiness probe                                                                  | See below       |
-| deployment.probes.livenessProbe| The livenessness probe                                                                 | See below       |
+| deployment.probes.readinessProbe.enabled | Enabled readiness probe                                                                  | true       |
+| deployment.probes.readinessProbe.failureThreshold | When a probe fails, Kubernetes will try failureThreshold times before giving up.                                                                  | 3      |
+| deployment.probes.readinessProbe.periodSeconds | Perform probe  everytime after specified periodSeconds                                                                  | 10       |
+| deployment.probes.readinessProbe.successThreshold | Minimum consecutive successes for the probe to be considered successful after having failed.                                                                  | 1       |
+| deployment.probes.readinessProbe.timeoutSeconds | Number of seconds after which the probe times out.                                                                  | 1       |
+| deployment.probes.readinessProbe.initialDelaySeconds | Number of seconds after the container has started before liveness or readiness probes are initiated.                                                                  | 10       |
+| deployment.probes.readinessProbe.httpGet | Describes an action based on HTTP Get requests                                                                  |   path: '/path' port: 8080     |
+| deployment.probes.readinessProbe.exec | Kubelet executes the specified command to perform the probe                                                                  |   {}   |
 
 ##### Readiness Probe
 
 ```
     readinessProbe:
+      enabled: true
```

Comment:
I think this can be removed from here. and add it in section below

Reply:
Its part of configuration so it should be same everywhere. Causes confusion

Reply:
I mean readinessProbe complete section should be removed

Reply:
whole section of it